pftools/python/parflow/tools/core.py

```diff
@@ -432,3 +459,19 @@ def dist(self, pfb_file, **kwargs):
         from parflowio.pyParflowio import PFData
         pfb_data = PFData(pfb_file_full_path)
         pfb_data.distFile(p, q, r, pfb_file_full_path)
+
+    def import_metadata(self):
+        """Imports any applicable metadata for the current run
+
+        For instance, if CLM is in use, the CLM driver files will be
+        imported.
+        """
+        MetadataBuilder(self).build()
+
+    def write_metadata(self):
```

Comment:
CLM is not a metadata per say. I'm not sure having CLM part of import/export metadata make sense.